packages/config/src/options.ts

```diff
@@ -4,6 +4,7 @@ import * as validate from './validation'
 import pkg from '@packages/root'
 
 export type BreakingOptionErrorKey =
+  | 'COMPONENT_FOLDER_DEPRICATION'
```

Comment:
Seems this should be named REMOVED instead of DEPRICATION. Deprication means it will be removed in a future version, but this is already removed.

Reply:
Renamed. Also renamed `TEST_FILES_DEPRECATION` to `TEST_FILES_RENAMED`, since it's similarly an error about invalid configs.